content/asset-tracking/using-the-sdks.textile

```diff
@@ -5,7 +5,7 @@ index: 20
 meta_description: "Ably Asset Tracking SDKs provide an easy way to track multiple assets with realtime location updates powered by the Ably realtime network and Mapbox Navigation SDK with location enhancement."
 meta_keywords: "Ably, Asset Tracking SDK, Mapbox, Mapbox Navigation SDK, Ably realtime, map matching, location tracking, Publishing SDK, Subscribing SDK"
 languages:
-  - android
+  - kotlin
```

Comment:
If we're going to change the content here, we should also do the same for index.textile and example-apps.textile as well.

Reply:
Done in [d976084](https://github.com/ably/docs/pull/1181/commits/d976084dbcbc6e4e048582d4cb1f81782e76e913)

Reply:
Thanks Mark - LGTM